utils/docker_build/Dockerfile

```diff
@@ -1,7 +1,10 @@
 # Copyright 2020 Intel Corporation
 # SPDX-License-Identifier: Apache 2.0
 
-FROM ubuntu:bionic
+FROM ubuntu:20.04
+
+ENV SAFESTRING_ROOT=/home/sdouser/safestringlib \
```

Comment:
Can be moved to a .env file as is done in the other components?

Reply:
Not required as this is container to build the component. For runtime containers, we should look at opportunities to move the configuration parameters to .env file.